### ISSUE TITLE
Failed to pull image  aspnet:4.8-windowsservercore-ltsc2016 on Windows Server 2016

### ISSUE THREAD

**Issue by KaisMarzouki** [docker/for-win#10760]
Hi everyone,
I can not pull mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-ltsc2016 image on Windows Server 2016 host machine.
![image](https://user-images.githubusercontent.com/20477743/112662052-846d8100-8e57-11eb-88cb-95e3ad8ef539.png)
**Used environment:**

1. Host machine OS Version: Microsoft Windows Server 2016 Datacenter 10.0.14393 N/A Build 14393
2. Docker Version:   20.10.0 
3. Docker  API version: 1.41 (minimum version 1.24)

When I try " **docker pull mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-ltsc2016**" command
I get the following error:
_failed to register layer: re-exec error: exit status 1: output: hcsshim::ProcessUtilityVMImage
 \\?\C:\ProgramData\docker\windowsfilter\afbe4420a54d638fb4852e7743c069a69415fb61dda0835b29822000b50cb942\UtilityVM:
The parameter is incorrect_

Thanks.

**Comment by stephen-turner** [comment]
Thanks for the report. This does not look like an issue with the Docker Desktop application itself but with the upstream docker windows container implementation, so could you please open an issue on https://github.com/moby/moby instead? Sorry, I would just move it myself, but it's not possible because that is not part of the docker org. Thank you.

**Comment by KaisMarzouki** [comment]
Hello Stephen,
First, thank you for your feedback.
There is good news! I have found a solution for my reported issue and I want to share it with the community.
The issue is related to Windows container version compatibility; the container image OS version and the Host OS version have the same OS Version (10.0.14393 N/A Build 14393) but there is a difference in the Revision number (patching).
Solution:
I have fixed the issue by installing the  Hotfix(s) in the host machine  (To download from [https://www.catalog.update.microsoft.com/](https://www.catalog.update.microsoft.com/)     
          
                           [01]: KB4601051
                           [02]: KB4132216
                           [03]: KB4486129
                           [04]: KB5001078
                           [05]: KB5000803 
Best regards
